android/README.md

```diff
@@ -0,0 +1,7 @@
+# OpenThread Commissioner Android App
+
+The OpenThread (OT) Commissioner sample App commissions a new Thread 1.1 Device into existing Thread Network.
```

Comment:
The OpenThread (OT) Commissioner Android App commissions a new Thread 1.1 Device onto an existing Thread Network.